src/shared/components/card/cardHistory/CardHistory.tsx

```
@@ -0,0 +1,24 @@
+
+interface CardHistoryProps extends React.ComponentProps<'div'> {
+  src: string;
+  title: string;
+  btnText: string;
+}
+
+const CardHistory = ({ src, title, btnText }: CardHistoryProps) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.imgbox}>
+        <img src={src} className={styles.imgbox} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)